documentation/src/pages/index.module.css

```
@@ -1,116 +1,537 @@
- * CSS files with the .module.css suffix will be treated as CSS modules
- * and scoped locally.
- */
+.homepage {
+    --home-logo-navy: #003c69;
+    --home-logo-blue: #3070b3;
+    --home-tum-blue: #0065bd;
+    --home-sky: #98c6ea;
+    --home-ice: #eaf4fb;
+    --home-ink: #102a3d;
+    --home-secondary-hover: rgba(152, 198, 234, 0.34);
+    display: flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)